docs/source/deployment/netlify.md

```diff
@@ -0,0 +1,188 @@
+3. No need to worry about scaling containers due to a serverless architecture.
+
+We'll set up a "Hello World" API and frontend for an Apollo app on Netlify. Let's get started!
+
+## Download and Run the Netlify Starter Kit
+
+Download and run the [official Netlify create-react-app and Lambda starter kit](https://github.com/netlify/create-react-app-lambda). This will set us up with a base for our frontend and API code.
+
+
+```bash
+git clone https://github.com/netlify/create-react-app-lambda.git
+cd create-react-app-lambda
+npm install
+```
+
+Let’s take a quick look at the file structure here:
+
+* **package.json:** our dependencies, shared between client and server code. You can also split them up into separate directories later, depending on your preference.
+* **netlify.toml:** configuration for Netlify and the Netlify functions.
+* **src/:** code for our React app.
+* **src/lambda/:** code that will be deployed to Lambda by Netlify.
+
+In `package.json`, you’ll find scripts to run the frontend and lambda code:
+
+```js
+"scripts": {
+  "start": "react-scripts start",
+  "start:lambda": "netlify-lambda serve src/lambda",
+  "build": "react-scripts build",
+  "build:lambda": "netlify-lambda build src/lambda",
+}
+```
+
+Run the code to see that everything is working. We have to open two terminal windows to run the frontend and functions at the same time:
+
+```bash
+# In the first terminal
+npm run start:lambda
+# In the second terminal
+npm start
+```
+
+If everything looks like it started correctly, let’s add a GraphQL API with Apollo Server!
+
+### Setting up Apollo Server
+
+Netlify functions run on AWS Lambda, so we can use the `apollo-server-lambda` package to easily integrate Apollo Server for our API layer. Let’s install the packages we need for that:
+
+```bash
+npm install --save apollo-server-lambda graphql
+```
+
+Now, we can create a “Hello world” GraphQL API. Let’s put that in a new file in the lambda folder, at `src/lambda/graphql.js`:
+
+```js
+// src/lambda/graphql.js
+const { ApolloServer, gql } = require("apollo-server-lambda");
+
+const typeDefs = gql`
+  type Query {
+    hello: String
+  }
+`;
+
+const resolvers = {
+  Query: {
+    hello: (root, args, context) => {
+      return "Hello, world!";
+    }
+  }
+};
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers
+});
+
+exports.handler = server.createHandler();
+```
+
+Now, make sure you’ve run `npm run start:lambda`, and navigate to `localhost:9000/graphql` in your browser. You should see GraphQL Playground, where you can run queries against your API!
+
+<div style="text-align:center">
+![Local GraphQL Server](../images/graphql.png)
+<br></br>
+</div>
+
+If you can see GraphQL Playground and run a simple query, you’ve done everything properly! It’s pretty neat that getting an API up and running was just a few lines of code. Now let’s add Apollo to our frontend.
+
+### Adding Apollo Client to React
```

Comment:
Hmm.  I haven't checked if we're doing this elsewhere in the server section, but I don't think demonstrating client details belongs in this guide.  By putting it here, it makes management of the documentation more difficult since those that are responsible for the Apollo Client changes won't be able to easily locate and fix these as client versions evolve.